### PR TITLE
[Infastructure] Re-enable template tests on master

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

File: src/ProjectTemplates/test/ProjectTemplates.Tests.csproj [hunk 1]
```diff
@@ -10,8 +10,8 @@
 
     <RunTemplateTests Condition="'$(RunTemplateTests)' == ''" >true</RunTemplateTests>
     <!-- Skip template tests temporarily while we investigate 5.0 migration issues -->
```
Comment (JunTaoLuo) [inline src/ProjectTemplates/test/ProjectTemplates.Tests.csproj]: remove this comment

**Review by JunTaoLuo — COMMENTED** [review]
Should rebase this on https://github.com/aspnet/AspNetCore/pull/14168.

**Review by dougbu — CHANGES_REQUESTED** [review]
Suggest closing this until I finish the (extra large) merge of the last few weeks' worth of commits from 'release/3.1'. All but two project template tests should be running in 'release/3.1' at the moment.

**Comment by javiercn** [comment]
> Suggest closing this until I finish the (extra large) merge of the last few weeks' worth of commits from 'release/3.1'. All but two project template tests should be running in 'release/3.1' at the moment.

I'm trying to capture a dump of whatever is causing this to hang. That's why its running. It's bad enough that these tests got disabled in the first place.

**Comment by javiercn** [comment]
This was done separately.

**Pull request closed without merging**